dataset/models/tf/base.py

```diff
@@ -617,6 +617,20 @@ def num_channels(self, tensor, **kwargs):
         channels_axis = self.channels_axis(tensor, **kwargs)
         return shape[channels_axis] if shape else None
 
+    @staticmethod
```

Comment:
why static? classmethod fits here, doesn't it?

Reply:
No, that method is used in linknet decoder_block so it must be static. In that branch I don't rename non-static methods into 'get_...'

Reply:
You're right.